scripts/generate-config.sh

```diff
@@ -39,12 +39,22 @@ compare_extensions=(
   yaml
   go
   proto_descriptor
+  html
 )
 
+exit_with=0
+echo "Generating config against istio $(cd ${istio_src} && git describe --contains --all HEAD)"
```

Comment:
The `--contains` options is using an upwards tag (a "later" tag/version) for describing the reference ("HEAD" in this case), which I find confusing and potentially erroneous: most of the time you will get the name of a future release tag, ie. 1.1.0 even if your code is 1.0.X based, or the branch name (using `--all`), which moves and does not help reproduce issues. I am not sure if you went with that for any particular property?

I would suggest using `git describe --dirty --tags --abbrev=12`, which bases the output on past releases + changes, states whether there is any modification, and adds a good chunk of the commit id (the reference `HEAD` is implied).